src/core/Makefile.am

```diff
@@ -31,13 +31,18 @@ include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
 lib_LIBRARIES                       = \
     libopenthread-ftd.a               \
     libopenthread-mtd.a               \
+    libopenthread-radio.a             \
     $(NULL)
 
 CPPFLAGS_COMMON                     = \
     -I$(top_srcdir)/include           \
     $(OPENTHREAD_TARGET_DEFINES)      \
     $(NULL)
 
+libopenthread_radio_a_CPPFLAGS      = \
+    $(CPPFLAGS_COMMON)                \
+    -DOPENTHREAD_RADIO=1              \
```

Comment:
I know you are following the `OPENTHREAD_MTD` and `OPENTRHEAD_FTD` naming here.. but `OPENTRHEAD_RADIO` as a config define seems too generic and not very descriptive... I don't have a better suggestion for what to name this (will think about it)...